p2p/__init__.py

```diff
@@ -602,44 +602,49 @@ def clone_content_item(self, slug, clone_slug, keep_embeds=False, keep_relateds=
                 }
                 payload['related_items'].append(related_item)
 
+        found_contributors = find_clone_contributors(content_item)
+
+        if 'contributors' in content_item:
```

Comment:
I don't think this works.  I don't think we can just shove a contributors list from one content item into another